telethon/client/uploads.py

```diff
@@ -72,7 +77,10 @@ def _resize_photo_if_needed(
             result.paste(image, mask=image.split()[alpha_index])
 
         buffer = io.BytesIO()
-        result.save(buffer, 'JPEG')
+        if exif:
+            result.save(buffer, 'JPEG', exif=exif)
+        else:
+            result.save(buffer, 'JPEG')
```

Comment:
If `save`'s `exif=` defaults to `None`, we already have a "maybe-`None`" value:

```suggestion
        result.save(buffer, 'JPEG', exif=exif)
```

Again, my suggested change needs testing.

Reply:
OK, I will test it.